packages/stores/src/account/base.ts

```diff
@@ -565,10 +565,10 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
         ...signOptions,
       };
 
-      let usedFee: StdFee;
-      if (typeof fee === "undefined") {
+      // Estimate gas fee & token if not provided
+      if (!fee) {
```

Comment:
Let's ensure this change doesn't break any of our existing features. Note that `typeof fee === "undefined"` is not equivalent to `!fee`